action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   slack_webhook_url:
     description: "A Slack incoming Webhook URL"
     required: true
+  build_release_artifacts:
+    description: "Run `make release_artifacts` and expect results in `release_artifacts/`"
```

Comment:
We could let the action caller create the artifact in any way they want and provide a directory from which to take it. To not impose the use of `make`.